Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -159,6 +160,18 @@ public IncrementalLoadingCollection(int itemsPerPage = 20, Action onStartLoading
         /// <seealso cref="IIncrementalSource{TSource}"/>
         public IncrementalLoadingCollection(TSource source, int itemsPerPage = 20, Action onStartLoading = null, Action onEndLoading = null, Action<Exception> onError = null)
```

Comment:
the original code used 2nd constructor primarly and I am continuing to use it. What I have changed is the param passed to 2nd constructor. This isn't a valid objection